examples/DotNetCore/AzureFunction/FunctionAppIsolatedMode/AzureAppConfigurationRefreshMiddleware.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Middleware;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+
+namespace FunctionAppIsolatedMode
+{
+    internal class AzureAppConfigurationRefreshMiddleware : IFunctionsWorkerMiddleware
+    {
+        public IEnumerable<IConfigurationRefresher> Refreshers { get; }
```

Comment:
Why is Refreshers public?

Reply:
No reason. I copied the code from the existing asp.net core middleware :). Updated to use private field.